network/src/primary.rs

```diff
@@ -245,7 +247,7 @@ impl PrimaryToWorkerNetwork {
             .await
     }
 
-    pub async fn broadcast<T: VerifyingKey>(
+    pub async fn unreliable_broadcast<T: VerifyingKey>(
```

Comment:
Similar, since you are doing the renaming , a short 1-2 lines of what I should expect from this method would be great!